### PR TITLE
Console v2 beta

### PR DESCRIPTION
**Pull request by jdgemm**

### CODE REVIEW

File: docs/use-the-network/console/console-v2-beta.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+id: console-v2-beta
+sidebar_label: FAQ
+slug: /use-the-network/console/console-v2-beta
+---
+
+# Console 2.0 Beta FAQ
+
+**Important** on staging the first-time a device joins could take up to 10 to 20 mins. This is because the device information is being added to the blockchain and needs to be propagated to miners.
```
Comment (PharkMillups) [inline docs/use-the-network/console/console-v2-beta.mdx]: I would make this whole section an info or warning box.

File: docs/use-the-network/console/console-v2-beta.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### What’s changed in Console 2.0?
+
+The biggest new feature of Console 2.0 is Flows. 
+Flows is a visual-centric view of the key elements in Console and allows users to instantly understand the relationship among devices, functions, and integration.
```
Comment (PharkMillups) [inline docs/use-the-network/console/console-v2-beta.mdx]:
```suggestion
Flows is a visual-centric view of the key elements in Console and allows users to instantly understand the relationship among devices, functions, and integrations. 
```

File: docs/use-the-network/console/console-v2-beta.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### What’s changed in Console 2.0?
+
+The biggest new feature of Console 2.0 is Flows. 
+Flows is a visual-centric view of the key elements in Console and allows users to instantly understand the relationship among devices, functions, and integration. 
+
+### What’s the same in Console 2.0?
+
+The interface and work flows for the following have remained largely unchanged:
+- Data Credit
```
Comment (PharkMillups) [inline docs/use-the-network/console/console-v2-beta.mdx]:
```suggestion
- Data Credits 
```

File: docs/use-the-network/console/console-v2-beta.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Provide feedback [here](https://github.com/helium/console/issues) and use label (staging).
+
+### Why did you make these changes to Console 2.0?
+
+Based on learnings from your feedback, the team wanted to deliver a much more streamlined and intuitive experience to accomplish core user goals including adding devices, defining functions, and setting up integrations to transfer data to endpoints.
+
+### Do labels still exist? Do I still need labels? What capabilities do labels have?
+
+Yes Labels will still exist to organize and manage devices. Users can choose to apply a single or multiple Labels  to as many devices as needed, for example, to identify a sensor type like a temp sensor or location of the sensor. Anytime Labels are updated the changes flow to all devices with the Labels.
```
Comment (PharkMillups) [inline docs/use-the-network/console/console-v2-beta.mdx]:
```suggestion
Yes. Labels will still exist to organize and manage devices. Users can choose to apply a single or multiple Labels  to as many devices as needed; for example, to identify a sensor type like a "temp sensor" or location of the sensor. Anytime Labels are updated the changes flow to all devices with the Labels.
```

File: docs/use-the-network/console/console-v2-beta.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+### What happens to functionality that used to be tied to labels (ADR, Notifications, multiple packet purchases)?
+
+This functionality still exists. These settings are defined as general settings and can be applied to devices or groups of devices in the Flows workspace. Documentation will be coming.
+
+### When will Console 2.0 get released?
+
+It depends on the feedback and issues reported during the Console 2.0 beta phase, but our goal is to release Console 2.0 in approximately 4 - 6 weeks.
```
Comment (PharkMillups) [inline docs/use-the-network/console/console-v2-beta.mdx]: 4-6 weeks from when? I would put an actual date in there.

File: docs/use-the-network/console/console.mdx [hunk 6]
```diff
@@ -18,8 +18,9 @@ Device Management includes:
 - Device ID registration and seamless, secure onboarding and authentication
 - Data Credits for utilizing the Helium Network
 
-Console is available as a free to use hosted application at
-[console.helium.com](https://console.helium.com).
+[Console 2.0 Beta](/use-the-network/console/console-v2-beta) is available for testing and feedback. 
+
+Console is available as a free to use hosted application at [console.helium.com](https://console.helium.com).
```
Comment (PharkMillups) [inline docs/use-the-network/console/console.mdx:25]: Make this entire section an info call out.

**Review by PharkMillups — CHANGES_REQUESTED** [review]
Left you a few comments.

File: docs/use-the-network/console/console.mdx (hunk 6 above)
Comment (jdgemm) [reply docs/use-the-network/console/console.mdx:25]: unclear which section

Comment (jdgemm) [reply docs/use-the-network/console/console.mdx:25]: nvm figured it out

**Review by PharkMillups — APPROVED** [review]

**Pull request merged**